test/benchmark/iperf3/tcp_bw/result_template.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "name": "Average Bandwidth between Host Linux and Guest Linux",
```

Comment:
The name is missing the `virtio-driver` and `TCP` information, but I think it's too long to add both. Thus, just adding `TCP` to the name is fine for now, since we only support the `virtio-net` driver.